include/nuttx/timers/capture.h

```diff
@@ -40,6 +40,11 @@
 #define CAPIOC_FREQUENCE _CAPIOC(2)
 #define CAPIOC_EDGES     _CAPIOC(3)
 #define CAPIOC_ALL       _CAPIOC(4)
+#define CAPIOC_PULSES    _CAPIOC(5)
```

Comment:
can you add callback to cap_ops_s or explain how each ioctl work and which argument it accepts?